src/test/regress/sql/pg14.sql

```diff
@@ -712,3 +712,10 @@ drop extension postgres_fdw cascade;
 drop schema pg14 cascade;
 reset client_min_messages;
 select 1 from citus_remove_node('localhost',:master_port);
+
+-- an example with CREATE TABLE LIKE, with statistics with expressions
+CREATE TABLE ctlt1 (a text CHECK (length(a) > 2) PRIMARY KEY, b text);
+CREATE STATISTICS ctlt1_expr_stat ON (a || b) FROM ctlt1;
+CREATE TABLE ctlt_all (LIKE ctlt1 INCLUDING ALL);
+SELECT create_distributed_table('ctlt1', 'a');
+CREATE TABLE ctlt_all_2 (LIKE ctlt1 INCLUDING ALL);
```

Comment:
This should probably be higher up in the file, before the pg14 schema gets dropped. Otherwise these tables will linger throughout other tests.